tests/DevApps/daemon-app/daemon-app-msi/appsettings.json

```diff
@@ -0,0 +1,18 @@
+{
+    "AzureKeyVault": {
+        "BaseUrl": "https://revoguardkeyvault.vault.azure.net/",
```

Comment:
I will also add a small readme to this dev app, on how to run this to get secret from the vault. Inside a VM assigned with MI, and the MI provided has access to the KV, this dev app will use IdWeb to get that secret.